### ISSUE TITLE
Formatting of stack traces

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#19]
Some stack traces show up as follows:
```
>>> Attempting to dereference a 'Nil' memory instance <<<
Unwinding local stack (0000015EA8C2DC80):
  method: pos=3, name='Evaluator->Evaluate(p0:Node)'
  method: pos=2, name='Evaluator:Evaluate:'
  method: pos=1, name='MiniLisp:Main:o.System.String*,'
```
Only the first list of the stack trace is pretty printed

**Comment by objeck** [comment]
Fixed in push 3f7d47d69

**Comment by objeck** [comment]
fixed in v5.7.2
